<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resultados</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background-color:rgb(255, 255, 0);}</string>
  </property>
  <widget class="QLabel" name="label_resultados">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>s</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(85, 255, 0);}

QPushButton:hover{background-color:orange;}</string>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_resultados_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Los atributos seleccionados fueron:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>231</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_kp1">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kp1: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_N">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>N:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_U">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>U: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_dp">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>dp:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_P">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>P: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_T">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>T:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_material">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Material:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Barra-de-carbon-Homer-simpson.jpg</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
